examples/modelzoo/evaluate_caffe_net.py

```diff
@@ -124,8 +122,7 @@ def forward(x, t):
         del x, y, loss, accuracy
 
         count += args.batchsize
-        print('{} / {}'.format(count, len(dataset)), end='\r', file=sys.stderr)
-        sys.stderr.flush()
+        print('{} / {}'.format(count, len(dataset)), end='\r')
```

Comment:
`end='\r'` also needs to be removed.

Reply:
Ah, it seems we should replace thsese `print`s with `sys.stdout.write`.